src/test/kotlin/no/nav/syfo/testutil/mock/ManglendeMedvirkningMock.kt

```diff
@@ -0,0 +1,57 @@
+package no.nav.syfo.testutil.mock
+
+import io.ktor.server.application.call
+import io.ktor.server.engine.*
+import io.ktor.server.netty.*
+import io.ktor.server.response.respond
+import io.ktor.server.routing.*
+import no.nav.syfo.personstatus.api.v2.auth.installContentNegotiation
+import no.nav.syfo.personstatus.application.manglendemedvirkning.ManglendeMedvirkningDTO
+import no.nav.syfo.personstatus.application.manglendemedvirkning.ManglendeMedvirkningResponseDTO
+import no.nav.syfo.personstatus.application.manglendemedvirkning.ManglendeMedvirkningVarselDTO
+import no.nav.syfo.personstatus.application.manglendemedvirkning.ManglendeMedvirkningVurderingType
+import no.nav.syfo.personstatus.infrastructure.clients.arbeidsuforhet.ManglendeMedvirkningClient
+import no.nav.syfo.testutil.UserConstants
+import no.nav.syfo.testutil.getRandomPort
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.*
+
+class ManglendeMedvirkningMock {
```

Comment:
Har forenklet litt oppsettet for disse mockene i siste versjon på master :)

Reply:
Tror jeg har klart å tilpasse PR'en nå.